bundle/Resources/config/services/layout_resolver.yaml

```diff
@@ -44,6 +44,25 @@ services:
         tags:
             - { name: netgen_layouts.target_type.doctrine_handler, target_type: sylius_taxon }
 
+    netgen_layouts.sylius.layout_resolver.target_type.page:
+        class: Netgen\Layouts\Sylius\Layout\Resolver\TargetType\Page
+        arguments:
+            - "%netgen_layouts.sylius.target.pages%"
+        tags:
+            - { name: netgen_layouts.target_type, priority: 170 }
```

Comment:
There's already a target type with 170 as priority, so this should be 140.

Reply:
@emodric Fixed in https://github.com/netgen-layouts/layouts-sylius/pull/14/commits/2405cdffda3e9e3b03a81c14ce552414239a41ae